pkg/registry/event/rest.go

```diff
@@ -45,7 +46,8 @@ func NewStorage(registry generic.Registry) *REST {
 func (rs *REST) Create(ctx api.Context, obj runtime.Object) (runtime.Object, error) {
 	event, ok := obj.(*api.Event)
 	if !ok {
-		return nil, fmt.Errorf("invalid object type")
+		glog.V(2).Infof("Invalid object type. Was expecting an object of type api.Event: %v", obj)
```

Comment:
Actually, if these errors are going to be returned to users, it would be nice to be in terms of user API concepts instead of Go code, which the existing error message does, though without information about what Kind was passed. So maybe we should leave it the way it is. It needs to be at a higher verbosity level, though, if it's caused by user error rather than bug.